rented-properties/src/main/java/org/egov/cpt/web/controllers/RentDemandGenerationController.java

```diff
@@ -31,7 +31,7 @@ public RentDemandGenerationController(RentDemandGenerationService demandGenerati
 	@PostMapping("/_create")
 	public ResponseEntity<?> create(@Valid @ModelAttribute RentDemandCriteria demandCriteria,
 			@Valid @RequestBody RequestInfoWrapper requestInfoWrapper) {
-		demandGenerationService.createDemand(demandCriteria);
```

Comment:
requestInfo may not be sent. as this is called as a url without body. This may not work. 

If, we need requestInfo to get the tenantId and roles. Can we hard code request info like how we did for the tests?

Reply:
requestinfo hard coded